### ISSUE TITLE
Prevent Start Menu Search field from being tiled

### ISSUE THREAD

**Issue by zolrath** [rickbutton/workspacer#96]
**Problem:**

When opening the start menu with the windows key the search menu takes up a tile and is offset from the start menu itself.

![image](https://user-images.githubusercontent.com/454563/83953315-66fdf680-a7f4-11ea-8083-151303cf18e0.png)

If I've switched to the full-screen workspace this causes my currently active application to minimize.
If I'm in a split workspace it takes up a standard tile space with the results of the search.

**Expectation:**

I would expect the start menu and its search bar to remain together and not split into separate tiles/windows.

I would expect the start menu to simply float above currently active programs and not change focus or be considered a tile of its own.

**Specs:**
Windows Version: 10.0.19041 Build 19041

**Comment by zolrath** [comment]
Adding:
```C#
    context.WindowRouter.AddFilter((window) => !window.ProcessFileName.ToLower().Contains("searchapp"));
```
was enough to correct this in the configuration, though it feels like it should be default?

**Comment by josteink** [comment]
> Adding:
> 
> ```C#
> 
>     context.WindowRouter.AddFilter((window) => !window.ProcessFileName.ToLower().Contains("searchapp"));
> 
> ```
> 
> was enough to correct this in the configuration, though it feels like it should be default?

Agreed. Make a PR for it? 🙂

**Comment by takase1121** [comment]
This introduced a problem with my computer. With the filter, my search is now 'clipped'.

![image](https://user-images.githubusercontent.com/20792268/84586867-6943fe80-ae4d-11ea-9ccd-5d05d456ea8f.png)

**Comment by takase1121** [comment]
btw, the filter line can be shortened to 
`context.WindowRouter.IgnoreProcessName("SearchApp");`

**Comment by rickbutton** [comment]
this will be fixed in `0.9.8`

**Comment by rickbutton** [comment]
`0.9.8` is released! workspacer should automatically prompt for the update.
